### DISCUSSION TITLE
Why do you chose to partiticpate in the GitHub experience?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23005] [New to GitHub]
<p>Hello GitHub Community Forum!</p>
<p>I am a student at San Jose State University, studying the information needs of the GitHub community. I am excited by the enthusiasm and professionalism of those working to make their coding projects the best possible. I am also excited to read the many posts in these community forums that ask insightful questions to solve specific issues. As part of my research, I am reaching out to members of the GitHub community to review questions pertinent to my research. Hopefully, this information will expand my current understanding of this ecosystem. Please feel free to comment on these questions. The answers will be gathered and studied.</p>
<p>What makes the GitHub ecosystem a community?</p>
<p>What makes the GitHub community unique?</p>
<p>What types of information do GitHub developers and coders need?</p>
<p>How do developers and coders find the information required?</p>
<p>What type/kind of information have you received through the community and was it of use?</p>
<p>Have I missed something?</p>
<p>Please feel free to comment on any or all of these questions.</p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/potomac64">@potomac64</a>,</p>
<p>Welcome to the Github Community Forum!</p>
<blockquote>
<p>What makes the GitHub ecosystem a community?</p>
</blockquote>
<p>This is a good question and is not so simple to explain. To me make possible to take part of develop something from the little project to the big one is awesome.</p>
<blockquote>
<p>What makes the GitHub community unique?</p>
</blockquote>
<p>The Github community is not unique, we need to be realistic. There are plenty of community that have the same pourpose and have more members. I don’t want to be misunderstood I love github and I’m proud to be an active member, in the other hand other community is necessary to have more tips to make this one greater. Hope I’ve made myself clear :slight_smile:</p>
<blockquote>
<p>What types of information do GitHub developers and coders need?</p>
</blockquote>
<p>Developers and coders are the same. The answer is easy: problems to solve, something that can prove the strength of the mind, more more and even more question to answer and so on. I think you’ve got the point.</p>
<blockquote>
<p>How do developers and coders find the information required?</p>
</blockquote>
<p>Experiences, mostly of the solution come from the method <strong>try and error</strong> , github community, stackoverflow, tutorial, pdf from internet, and in the worst scenario from books. Don’t get me wrong, I love books, but when you need a solution to a problem you never find on it.</p>
<blockquote>
<p> What type/kind of information have you received through the community and was it of use?</p>
</blockquote>
<p>Don’t remember.</p>
<blockquote>
<p>Have I missed something?</p>
</blockquote>
<p> If you have we are here.</p>
<p>-Gabriele-</p>
<p>Mark helpfull posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great contents!</p>
